src/gps/GPS.cpp

```diff
@@ -1313,7 +1314,10 @@ int32_t GPS::disable()
     enabled = false;
     setInterval(INT32_MAX);
     setAwake(false);
-
+    // reset position data if location is disabled and not fixed
+    if (config.position.gps_mode == meshtastic_Config_PositionConfig_GpsMode_DISABLED && !config.position.fixed_position) {
+        positionModule->clearLastPosition();
```

Comment:
I think this one is still valid even if the "gpsmode" is used for the hardware GNSS module, as when it's decativated there is no use for the previous data if "fixed position" is disabled.

Reply:
I would prefer a clean pull, probably doing this on nodedb reset.  @jp-bennett I think this could be related to the preventing a node from deleting itself from the nodedb, should we clear the last position from the current node when we reset the nodedb?

Reply:
> Does that change apply when doing a NodeDB reset?

The change i did applies only when GPS::disable() is called.